src/test/java/net/openhft/chronicle/bytes/BytesTest.java

```diff
@@ -1071,21 +1072,158 @@ public void isClear() {
         bytes.releaseLast();
     }
 
+    @Test
+    public void testAppendDouble() {
+
+        // TODO FIX
```

Comment:
please fix!

Reply:
This due to parseDouble() so needs another issue

Reply:
https://github.com/OpenHFT/Chronicle-Bytes/issues/527